content/en/docs/reference/api/account-api/_index.md

```diff
@@ -0,0 +1,508 @@
+
+```json
+[
+  {
+    "name": "Nickname",
+    "value": "Test"
+  }
+]
+```
+
+## PUT `/iam/account/{id}/attributes`
+
+Adds attributes to the user account. It can be done directly through the IAM dashboard by clicking on "Set attribute" button in *Attributes* section of the user homepage.
+
+Requires `ROLE_ADMIN`.
+
+```bash
+$ curl -X PUT -H "Accept: application/json" \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer ${AT}" -d @attribute.json \
+  http://localhost:8080/iam/account/80e5fb8d-b7c8-451a-89ba-346ae278a66f/attributes
+```
+
+where ```attribute.json``` is:
+
+```json
+{
+   "name":"Nickname",
+   "value":"Test"
+}
+```
+
+## DELETE `/iam/account/{id}/attributes`
+
+Deletes user attribute by adding the query parameter at the end of the endpoint.
+
+Requires `ROLE_ADMIN`.
+
+```bash
+$ curl -X DELETE -H "Authorization: Bearer ${AT}" \
+  http://localhost:8080/iam/account/80e5fb8d-b7c8-451a-89ba-346ae278a66f/attributes?name=Nickname
+```
+
+## GET `/iam/account/{id}/authorities`
+
+Retrieves user roles.
+
+Requires `ROLE_ADMIN`.
+
+    GET http://localhost:8080/iam/account/80e5fb8d-b7c8-451a-89ba-346ae278a66f/authorities
+
+```json
+{
+  "authorities": [
+    "ROLE_GM:c617d586-54e6-411d-8e38-649677980001",
+    "ROLE_USER"
+  ]
+}
+```
+
+The above example shows that the Test User is also a group manager (of group with ```c617d586-54e6-411d-8e38-649677980001``` id). 
+Remember that there are three roles in Indigo IAM: Amdin, User, Group Manager.
+
+## POST `/iam/account/{id}/authorities`
+
+Assignes an authority to the user.
+
+Requires `ROLE_ADMIN`.
+
+```bash
+$ curl -X POST -H "Content-Type: application/x-www-form-urlencoded" \
+  -H "Authorization: Bearer ${AT}" -d "authority=ROLE_ADMIN" \
+  http://localhost:8080/iam/account/80e5fb8d-b7c8-451a-89ba-346ae278a66f/authorities
+```
+
+## DELETE `/iam/account/{id}/authorities`
+
+Revokes user authority by specifying the query parameter.
+
+Requires `ROLE_ADMIN`.
+
+```bash
+$ curl -X DELETE -H "Authorization: Bearer ${AT}" \
+  http://localhost:8080/iam/account/80e5fb8d-b7c8-451a-89ba-346ae278a66f/authorities?authority=ROLE_ADMIN
+```
+
+## GET `/iam/account/me/clients`
+
+Retrieves information about clients owned by the currently authenticated user.
+
+    GET http://localhost:8080/iam/account/me/clients
+
+```json
+{
+  "totalResults": 1,
+  "itemsPerPage": 1,
+  "startIndex": 1,
+  "Resources": [
+    {
+      "client_id": "client",
+      "client_name": "Test Client",
+      "redirect_uris": [
+        "https://iam.local.io/iam-test-client/openid_connect_login",
+        "http://localhost:9090/iam-test-client/openid_connect_login"
+      ],
+      "scope": "address phone openid profile offline_access read-tasks attr write-tasks email read:/ write:/",
+      "created_at": 1658764288643
+    }
+  ]
+}
+```
+
+## GET `/iam/account/find/{options}`
+
+Filters user information by label, email, username, certificate subject or group. 
```

Comment:
Specify what are the actual available options and the actual corresponding attributes to use. For example, for the certificate subject the option is `bycertsubject` and the attribute is `certificateSubject`; moreover the value has to be URL-encoded. As an example:
```sh
curl -H "Authorization: Bearer $AT" "https://${IAM_HOST}/iam/account/find/bycertsubject?certificateSubject=CN%3dFrancesco%20Giacomini%20giaco%40infn.it%2cO%3dIstituto%20Nazionale%20di%20Fisica%20Nucleare%2cC%3dIT%2cDC%3dtcs%2cDC%3dterena%2cDC=org"
```